pyresample/utils.py

```diff
@@ -298,6 +298,110 @@ def get_area_def(area_id, area_name, proj_id, proj4_args, x_size, y_size,
                           x_size, y_size, area_extent)
 
 
+def _get_area_def_from_gdal(dataset, area_id=None, name=None, proj_id=None,
+                            proj_dict=None):
```

Comment:
Our maximum line width is 120. You could probably put all the arguments on the same line.